src/storage/gc_worker.rs

```diff
@@ -817,27 +817,35 @@ impl<S: GCSafePointProvider, R: RegionInfoProvider> GCManager<S, R> {
     /// example, when we just starts to do GC, our progress is like this: ('^' means our current
     /// progress)
     ///
-    ///      | region 1 | region 2 | region 3| region 4 | region 5 | region 6 |
-    ///      ^
+    /// ```text
+    ///     | region 1 | region 2 | region 3| region 4 | region 5 | region 6 |
```

Comment:
Maybe you can omit the leading space.

Reply:
Another way to accomplish this would be with blockquotes like

```
>   | region 1 ...
```

I don't have any preference though.

Reply:
Apart from removing leading spaces, I think the ` ```text ` is better to be kept, because if we consider the HTML rendered by this, the text out of code blocks may be rendered to non-monospace font
@overvenus How do you think?

Reply:
Thanks for the reminder, both look good to me.